pkg/az/az_test.go

```diff
@@ -70,7 +70,7 @@ var _ = Describe("AZ", func() {
 				})
 
 				It("should have returned 3 availability zones", func() {
-					Expect(len(selectedZones)).To(Equal(3))
+					Expect(len(selectedZones)).To(Equal(2))
```

Comment:
So I think we will basically need to expand the test to cover `DefaultRequiredAvailabilityZones`, and `MinRequiredAvailabilityZones` (as special case for us-east-1).